recipes/orthofinder/meta.yaml

```
@@ -18,7 +18,7 @@ requirements:
     - python >=3.7
     - scipy
     - blast
-    - diamond 
+    - diamond <2.1 
```

Wondering if a comment like: 

```suggestion
    # https://github.com/davidemms/OrthoFinder/issues/826 https://github.com/bbuchfink/diamond/issues/707
    - diamond <2.1
```

might be a good idea.